sandbox_request/pubsub.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2021 Universität Tübingen, DKFZ and EMBL
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Defines all async pub/sub communication"""
+
+import pika
+from ghga_service_chassis_lib.pubsub import AmqpTopic
+from .config import get_config
+
+
+def get_connection_params():
+    """Return a configuration object for pika"""
+    config = get_config()
```

Comment:
```suggestion
def get_connection_params(config):
    """Return a configuration object for pika"""
```

I'd suggest passing on the config object. Calling the get_config function twice feels a bit redundant

Reply:
Actually that is fine and it was designed for that (FastAPI will also call it upon every request):
The result is lru_cached, so even if you call it multiple times, it's really just executed once ;)

Reply:
The template and other repos use what I wrote. If we are going to change it in this one, we should probably follow through with the other repos that already have the implementation I followed. @MoritzHahn1337 @KerstenBreuer. Up to you to decide which one to use. We should try to just stick to one implementation only.

Reply:
Well, in that case just stick with what Kersten wrote.